packages/experimental/Switch/src/Switch.win32.tsx

```diff
@@ -47,15 +99,17 @@ export const Switch = compose<SwitchType>({
       const displayOnOffText = !!offText || !!onText;
       const isReduceMotionEnabled = AccessibilityInfo.isReduceMotionEnabled;
       const thumbAnimation = isReduceMotionEnabled ? { animationClass: 'Ribbon_SwitchThumb' } : null;
+      const currentOpacity = onOffTextTest ? 0 : 1; // hides the control during measurements
+      const newMinWidth = onOffTextTest ? null : { minWidth: finalWidth };
 
       return (
-        <Slots.root {...mergedProps}>
+        <Slots.root {...mergedProps} style={[{ opacity: currentOpacity }]}>
```

Comment:
Instead of passing in a `style` prop here, shouldn't we instead put this style in the .styling file?

Reply:
Actually, doing that change (at least the way I did it), prevented the opacity from properly working. I'll look into doing this change in a better way.

Reply:
This will fail a rerender test, so it should be memoized.

Reply:
Here's an example of a rerender test: [https://github.com/microsoft/fluentui-react-native/blob/a077890983b969cec18bfc138917a7499f46ef5b/packages/components/Button/src/Button.test.tsx#L66](https://github.com/microsoft/fluentui-react-native/blob/a077890983b969cec18bfc138917a7499f46ef5b/packages/components/Button/src/Button.test.tsx#L66)

Reply:
Changed implementation.